GIT_WORKFLOW.md

```diff
@@ -1 +1,40 @@
-Write based on: https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow
\ No newline at end of file
+# Git Workflow
+
+The Economist Digital Solutions teams use git for version control following the [Forking  Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow).
+
+The advantage of the Forking Workflow is that contributions can be integrated without the need for everybody to push to a single central repository. Developers push to their own repositories, and only maintainer teams can push to the official repository. This allows  maintainers to accept commits from any developer without giving them write access to the official codebase. This approach scales easily and encourages sharing and collaboration on code bases across teams with minimal overhead required to manage merging and branching.
+
+## Forking Workflow Overview
+
+With the Forking Workflow every developer has their own private repository instead of one central repo. This means that each developer has two git repositories for a single project: a personal one and a public one.
+
+## Setting Up Your Fork
+
+When you're ready to collaborate on a project, rather than cloning the project, create a Fork of the project. Github provide a link to create a Fork on the main page of the project's Github Repo. This creates a person version of the repo associated with your account only, where you can control who can push to it.
+
+After creating your fork, perform a git clone of the Fork to get a copy onto your local machine. This serves as your private development environment, just like in the other workflows.
```

Comment:
Are we seeing the devs we get would need this. I think the main process we are trying to follow is that Fork -> Ticket -> PR -> Review (Devs job to Follow) -> Merge.

Reply:
Yea that's the general flow.